R/individuals.R

```diff
@@ -196,45 +179,30 @@ create_variables <- function(parameters) {
   infectivity_values[subpatent] <- parameters$cu
 
   # Initialise the infectivity variable
-  infectivity <- individual::Variable$new("infectivity", function(n) infectivity_values)
+  infectivity <- individual::Variable$new("infectivity", infectivity_values)
 
-  drug <- individual::Variable$new("drug", function(n) rep(0, n))
-  drug_time <- individual::Variable$new("drug_time", function(n) rep(-1, n))
+  drug <- individual::Variable$new("drug", rep(0, size))
+  drug_time <- individual::Variable$new("drug_time", rep(-1, size))
 
-  rtss_vaccinated <- individual::Variable$new(
-    "rtss_vaccinated",
-    function(n) rep(-1, n)
-  )
-
-  rtss_boosted <- individual::Variable$new(
-    "rtss_boosted",
-    function(n) rep(-1, n)
-  )
+  rtss_vaccinated <- individual::Variable$new("rtss_vaccinated", rep(-1, size))
+  rtss_boosted <- individual::Variable$new("rtss_boosted", rep(-1, size))
 
   rtss_cs <- individual::Variable$new(
     "rtss_cs",
-    function(n) {
-      exp(parameters$rtss_cs[[1]] + parameters$rtss_cs[[2]] * rnorm(n))
-    }
+    exp(parameters$rtss_cs[[1]] + parameters$rtss_cs[[2]] * rnorm(size))
```

Comment:
Somewhat separate from this PR since you're just moving code around exists already but normally for `a + b * rnorm(n)` you'd right `rnorm(n, a, b)`. In fact:

```
> set.seed(1)
> a <- rnorm(100, sd = 10)
> set.seed(1)
> b <- 10 * rnorm(100)
> identical(a, b)
[1] TRUE
```

(impacts a number of copies of this below). This works for vectorised mean/sd